doc/modules/cross_validation.rst

```diff
@@ -235,6 +235,20 @@ Here is an example of ``cross_validate`` using a single metric::
     >>> sorted(scores.keys())
     ['estimator', 'fit_time', 'score_time', 'test_score', 'train_score']
 
+Cross-validation significance evaluation
+----------------------------------------
+
+Significance of cross validation scores can be evaluated using
+:func:`permutation_test_score` function. The function returns a p-value, which
+approximates the probability that the average cross-validation score would be
+obtained at by chance.
```

Comment:
```suggestion
obtained by chance if the target was independent of the data.
```

Reply:
@vmanisha committed the suggested changes to https://github.com/scikit-learn/scikit-learn/pull/14769